package.json

```diff
@@ -118,13 +118,18 @@
       "acorn-walk": {
         "peerDependencies": {
           "acorn": "*"
+        },
+        "peerDependenciesMeta": {
+          "acorn": {
+            "optional": true
+          }
```

Comment:
Added this since Vitest v1 relies on `acorn-walk` and has this peer dep warning. The `acorn` dep in `acorn-walk` is only used for types.